app/views/user/_posts.html.erb

```diff
@@ -0,0 +1,13 @@
+<% posts = [] %>
+<% current_user.posts.each do |post| %>
+  <% posts << post %>
+<% end %>
+<% posts.each do |post| %>
+  <div class="index-posts">
+    <p class='post-content'><%= post.content %></p>
```

Comment:
- [ ] the author of the post should be displayed as well, [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts4.png)